INSTALL.md

```diff
@@ -93,6 +93,23 @@ The [Releases](https://github.com/neovim/neovim/releases) page provides pre-buil
 
 ## Linux
 
+### Pre-built archives
+
+The [Releases](https://github.com/neovim/neovim/releases) page provides pre-built binaries for Linux systems. There are a few ways to install this, the way listed below can be changed to a different directory than the one used here (e.g. `$HOME/.local/bin ` if you do not have root privileges)
+
+```sh
+curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim-linux64.tar.gz
+# this is to clean up the files if you have previously installed it here before
+sudo rm -rf /opt/nvim
+sudo tar -C /opt -xzf nvim-linux64.tar.gz
+```
+
+After this you will need to tell your terminal where to find the `nvim` executable. Add the following line to your `~/.bashrc`:
+
+    export PATH="$PATH:/opt/nvim-linux64.tar.gz"
```

Comment:
You are not appending the correct path of the uncompressed tar, it should not include the `.tar.gz` extension

Reply:
Ah yes, you're right, dunno how I missed that. I'll fix that, thank you